### DISCUSSION TITLE
serverQueryContent body content for rss feed

### DISCUSSION THREAD

**Discussion by mhinton** [nuxt/content#1221] [General]
I am working on an rss feed and it doesn't seem like the result of a serverQueryContent is able to return the full content of the document in an easy way. Since doc.body is an object is there another property that will return the full body content as a string?

```javascript
import { serverQueryContent } from "#content/server";
import rss from "rss";

export default defineEventHandler(async (event) => {
  // Fetch all documents
  const docs = await serverQueryContent(event).find();
  const feed = new rss({
    title: "My Dev Notes",
    description:
      "A collection of notes on software development from me",
    feed_url: "https://example.dev/feed.xml",
    site_url: "https://example.dev",
    ttl: 1440,
  });
  for (const doc of docs) {
    if (!doc._draft) {
      feed.item({
        title: doc.title ?? "Untitled",
        description: doc.description ?? "body not found",
        url: doc._path ?? "https://example.dev/blog/",
        guid: doc._path,
        date: doc.date,
      });
    }
  }
  return feed.xml();
});

```

**Comment by nozomuikuta** [comment]
Duplicate of #1177 (not yet resolved as of 16th Jul 2022)

**Comment by nozomuikuta** [comment]
@mhinton 

I updated #1177 and there seems to be a solution for your issue.
It's appreciated if you close this thread so that Discussions list can be kept clean.

**Reply by mhinton** [reply]
This is a discussion not an issue and there doesn't appear to be a way to close these. 

See https://github.com/orgs/github-community/discussions/3097.
